### ISSUE TITLE
Why does Alertmanager require X-Scope-OrgID when auth_enabled is false?

### ISSUE THREAD

**Issue by jakubgs** [cortexproject/cortex#3396]
According to the docs:
```
# Set to false to disable auth.
# CLI flag: -auth.enabled
[auth_enabled: <boolean> | default = true]
```
And also:
https://github.com/cortexproject/cortex/blob/e4a8f3c289e263a9c49bfdddeeb59fa453c9a5af/docs/configuration/single-process-config.yaml#L6-L9
But when I set `auth_enabled: false` when running the `alertmanager` service I still am forced to provide `X-Scope-OrgID`:
```
 > curl -s http://localhost:9101/api/prom/configs/alertmanager 
no org id

 > curl -s http://localhost:9101/api/prom/configs/alertmanager -H 'X-Scope-OrgID: 0' 
the Alertmanager is not configured
```
Why?

**Comment by jakubgs** [comment]
Apparently this is already being worked on in https://github.com/cortexproject/cortex/pull/3343.

**Comment by gotjosh** [comment]
This is a duplicate of #3300 and #3260

Can be closed @pracucci

**Comment by pracucci** [comment]
Closed by #3343
